llama_stack/core/library_client.py

```diff
@@ -7,7 +7,7 @@
 import asyncio
 import inspect
 import json
-import logging
+import logging  # allow-direct-logging
```

Comment:
@leseb how should `_remove_root_logger_handlers` be implemented using `llama_stack.log.get_logger`?